xija/get_model_spec.py

```diff
@@ -42,8 +42,8 @@ def temp_directory():
     shutil.rmtree(tmpdir, ignore_errors=True)
 
 
-def get_xija_model_spec(model_name, version=None, repo_path=REPO_PATH,
-                        check_version=False, timeout=5) -> dict:
+def get_xija_model_spec(model_name, version=None, repo_path=None,
+                        check_version=False, timeout=5) -> tuple:
```

Comment:
Is this a bug fix (correcting the annotated return type)?  I originally though that you were changing the return type of the function, but now I don't see that in the diff.  same for Line 95.

Reply:
Yes, this a bug fix since it already returns a tuple.